wiki/Community/Reddit/en.md

```diff
@@ -0,0 +1,40 @@
+# Reddit
+
+This article lists places on [Reddit](https://reddit.com) where users are able to interact with osu! and its community.
+
+## Gameplay
+
+These unofficial community-driven subreddits focus mainly on gameplay topics.
+
+| Avatar | Name | Description |
+| :-: | :-: | :-- |
+| ![osugame avatar](img/osugame.png) | [/r/osugame](https://reddit.com/r/osugame) | **/r/osugame** is currently the largest multi-mode forum open for general discussions and score showcases. |
+| *None* | [/r/osumania](https://reddit.com/r/osumania) | **/r/osumania** offers a safe space specifically intended for the [osu!mania](/wiki/Game_mode/osu!mania) game mode. |
+
+## Mapping
```

Comment:
too many sections i think? just have a single list if each section is just a single reddit link.

Reply:
Here is a comparison with both versions:

| Single table | Separate sections |
| :-: | :-: |
| ![grafik](https://github.com/user-attachments/assets/c7bd4ebc-02c7-4a79-9973-adc40457096b) | ![grafik](https://github.com/user-attachments/assets/ad2af0ab-b4b7-4ab1-9d3f-62c7cdf5bcba) |